lib/index.js

```diff
@@ -204,6 +204,8 @@ function render(tree, options) {
         }
 
         result += html(node.content);
+      } else if (node._withoutClosingTag) {
```

Comment:
`_withoutClosingTag` The name is not very suitable because tag means that there is an opening and closing element.

It seems more appropriate to me that the `_closingElement`

Reply:
Regarding semantics, `element` in HTML refers to the whole thing - an element has start/opening and end/closing _tags_.

I agree double negation through the use of `without` in the name is unnecessary, but if a name such as `_closingTag` creates confusion, maybe we could use something like `_needsClosing` or `_shouldClose` or something similar?

Reply:
> Regarding semantics, `element` in HTML refers to the whole thing - an element has start/opening and end/closing _tags_.

It is considered that if the element is single such as `<hr>` or `<img>` then it is the `element` if the element has a closing element then the `tag` is used